### PR TITLE
Add feature flag for links to the water abstraction system

### PR DESCRIPTION
**Pull request by jonathangoulding**
This service is slowly being replaced. There are currently links directing the pages to the legacy licence view pages, built previously in the water-abstraction-ui repo.

We have rebuilt these pages (with some tweaks) in the water-abstraction-system repo with the intention of using the water-abstraction-system to view and manage licences.

This change will allow us to navigate to the water-abstraction-system licence view pages instead of the 'legacy' licence view pages in a controlled manner.

This change will be mirrored in water-abstraction-system.

https://eaflood.atlassian.net/browse/WATER-4466

### CODE REVIEW

File: src/external/views/nunjucks/view-licences/licences.njk [hunk 1]
```diff
@@ -104,10 +104,17 @@
           {% for licence in licenceData %}
             <div class="licence-result">
               <h2 class="licence-result__column">
-                <a href="/licences/{{licence.document_id}}">
-                  <span class="govuk-visually-hidden">View licence </span>
-                  {{ licence.system_external_id }}
-                </a>
+                {% if featureFlags.enableSystemLicenceView == true%}
```
Comment (jonathangoulding) [inline src/external/views/nunjucks/view-licences/licences.njk:107]: should only be internal ?

File: src/external/views/nunjucks/returns/macros/return-header.njk [hunk 2]
```diff
@@ -10,7 +10,11 @@
   </dl>
 
   <p class="govuk-!-margin-bottom-7">
-    <a href="/licences/{{ documentHeader.document_id }}">View this licence</a>
```
Comment (jonathangoulding) [inline src/external/views/nunjucks/returns/macros/return-header.njk:13]: should only be internal ?

**Comment by jonathangoulding** [comment]
If you change any of these in the nunjucks
```
    <li class="govuk-tabs__list-item">
            <a class="govuk-tabs__tab" href="#returns">Returns</a>
          </li>
```

The macro / js will stop working so that have been left alone. 

All the edges into this page should have been captured.

File: src/internal/config.js [hunk 3]
```diff
@@ -173,7 +173,8 @@ module.exports = {
     showReturnRequirements: (get(process.env, 'SHOW_RETURN_REQUIREMENTS') || '').toLowerCase() === 'true' && !isProduction,
     triggerSrocAnnual: (get(process.env, 'TRIGGER_SROC_ANNUAL') || '').toLowerCase() === 'true',
     useNewBillRunSetup: (get(process.env, 'USE_NEW_BILL_RUN_SETUP') || '').toLowerCase() === 'true',
-    useWorkflowSetupLinks: (get(process.env, 'USE_WORKFLOW_SETUP_LINKS') || 'true').toLowerCase() === 'true'
+    useWorkflowSetupLinks: (get(process.env, 'USE_WORKFLOW_SETUP_LINKS') || 'true').toLowerCase() === 'true',
+    enableSystemLicenceView: process.env.ENABLE_SYSTEM_LICENCE_VIEW === 'true'
```
Comment (jonathangoulding) [inline src/internal/config.js:177]: There is a mix of ways to check this is 'true'. Can change if needed.

**Review by Cruikshanks — APPROVED** [review]

**Pull request merged**